website/versions/source-slack.json

```diff
@@ -0,0 +1 @@
+{ "latest": "plugins-source-slack-v1.0.0" }
```

Comment:
Apparently this is unneeded and would be created by the our CI per @erezrokah 's comment [here](https://github.com/cloudquery/cloudquery/pull/5676#discussion_r1052118194).

Reply:
Yeah, maybe something changed, but initially the lack of this file was causing the build to fail. I can remove it and check again